README.md

```diff
@@ -1,19 +1,25 @@
 # Transifex Native iOS SDK
 
-Transifex Native iOS SDK is a collection of tools to easily localize your iOS applications using [Transifex Native](https://www.transifex.com/native/). The can fetch translations over the air (OTA) to your apps. It supports apps built both on Objective-C and Swift.
+Transifex Native iOS SDK is a collection of tools to easily localize your iOS applications 
+using [Transifex Native](https://www.transifex.com/native/). The can fetch translations 
```

Comment:
Just noticed this: `The can fetch` needs some kind of rephrasing, doesn't compute 🙂

Reply:
Fixed!